main.go

```diff
@@ -30,14 +30,7 @@ func main() {
 		sig := <-sc
 		fmt.Printf("\nGot signal [%v] to exit.\n", sig)
 		log.Warn("received signal to exit", zap.Stringer("signal", sig))
-		switch sig {
-		case syscall.SIGTERM:
-			cancel()
-			os.Exit(0)
-		default:
-			cancel()
-			os.Exit(1)
-		}
+		cancel()
```

Comment:
What about trying to graceful shutdown at the first time BR receives signal, and os.Exit() when it receives signal twice? So we can avoid hard code 30 seconds timeout.

Reply:
Good idea!

Reply:
Done, PTAL~